test/AutoDiff/generics.swift

```diff
@@ -57,25 +57,76 @@ struct SupervisedTrainer<Model : Layer> {
 }
 
 // Tests TF-440.
-struct TF_440_Input<Input: Differentiable, State: Differentiable>: Differentiable {
-    var input: Input
-    var state: State
+struct TF_440_Input<Input: Differentiable, State: Differentiable>
+  : Differentiable {
+  var input: Input
+  var state: State
 }
 struct TF_440<T : Differentiable> {
-    @differentiable
-    func applied(to input: TF_440_Input<Float, Float>) -> Float {
-        return input.state
-    }
-
-    @differentiable
-    func applied(to input: TF_440_Input<T, Float>) -> Float {
-        return input.state
-    }
-
-    @differentiable
-    func applied(to input: TF_440_Input<T, Float>) -> T {
-        return input.input
-    }
+  @differentiable
+  func applied(to input: TF_440_Input<Float, Float>) -> Float {
```

Comment:
Spacing was 4 and not in the 80 column limit, so I thought I'd update it here right away. Let me know if this is fine or if I should pull it out into a separate PR.